app/assets/stylesheets/ama_layout/layout_components/call-to-action.scss

```diff
@@ -0,0 +1,33 @@
+.call-to-action{
+
+  &__container{
+    @include grid-row;
+    @include clearfix;
+  }
+
+  &__sub-head{
```

Comment:
since the styles for this are the same as section__sub-head I don't think you need to create a new class here, just use section__sub-head